libs/sdk-ui-kit/api/sdk-ui-kit.api.md

```diff
@@ -7,8 +7,11 @@
 import { Component } from 'react';
 import { CSSProperties } from 'react';
 import { guidFor } from '@gooddata/goodstrap/lib/core/Guid';
+import { IFeatureFlags } from '@gooddata/goodstrap/lib/@next/Header/types';
+import { IHeaderMenuItem as IHeaderMenuItem_2 } from '@gooddata/goodstrap/lib/@next';
```

Comment:
this interface should be already in UI SDK

Reply:
I used in sdk instead of creating it thanks